src/shared/approval-state/get-approval-state.js

```diff
@@ -59,11 +59,11 @@ const getApprovalStateText = approvalState => {
     }
 }
 
-const getTagDisplayData = approvalState => {
+const getApprovalStateDisplayData = approvalState => {
```

Comment:
In some places we refer to "approval **state**" and in others "approval **status**". Which term do we want to settle on?

Reply:
Yeah I noticed that too. Status is from the design docs, and state from the API response. I think we should settle on `status` mostly because `state` is a term used very heavily by React. If we would opt for using `status` we'd avoid naming collisions.